tests/unit/PartialSubmissionControllerTest.php

```diff
@@ -53,6 +56,23 @@ public function testSavePartialSubmissionFieldCreated()
         $this->assertCount(1, $fields);
     }
 
+    public function testSavePartialSubmissionFileField()
+    {
+        $form = $this->objFromFixture(UserDefinedForm::class, 'form1');
+        $form->publishRecursive();
+        $field = $this->objFromFixture(EditableFileField::class, 'filefield1');
+        $field->write();
+        $multipart = [
+            'name'     => 'file1',
+            'contents' => fopen(__DIR__ . '/../fixtures/Hans-fullsize-sqr.png', 'rb'),
+            'filename' => 'Hans-fullsize-sqr.png'
+        ];
+
+        $response = Director::test('/partialuserform/save', ['File' => $multipart], [], 'POST');
```

Comment:
Any reason why you are not using `$this->savePartial()` or `$this->post()`?

Reply:
Pure habit